src/kernel/arch/aarch64/boot.zig

```diff
@@ -8,32 +9,41 @@ extern var KERNEL_STACK_END: *usize;
 
 export fn _start() linksection(".text.boot") callconv(.Naked) noreturn {
     // Halt all cores other than the primary core, until the kernel has multicore support
-    const core_id = asm ("mrs %[res], mpidr_el1"
-        : [res] "=r" (-> usize)
-    ) & 3;
-    if (core_id != 0) {
-        while (true) {
-            asm volatile ("wfe");
-        }
-    }
+    asm volatile (
```

Comment:
Believe it or not the zig code for testing which core requires using the stack, but the stack is not yet initialized. Thus done in asm instead.